src/brevitas/graph/equalize.py

```diff
@@ -107,15 +116,46 @@ def _get_input_axis(module):
 
 
 def _get_output_axis(module):
-    if isinstance(module, (torch.nn.Linear, torch.nn.Conv1d, torch.nn.Conv2d, torch.nn.Conv3d)):
+    if isinstance(module, (torch.nn.Linear, torch.nn.Conv1d, torch.nn.Conv2d, torch.nn.Conv3d, torch.nn.BatchNorm1d, torch.nn.BatchNorm2d, torch.nn.BatchNorm3d)):
         return 0
     elif isinstance(module, (torch.nn.ConvTranspose1d, torch.nn.ConvTranspose2d, torch.nn.ConvTranpose3d)):
         return 1
     else:
         raise RuntimeError(f"Module {module} not supported.")
 
 
-def _cross_layer_equalization(srcs, sinks):
+def _combine_weights_bias(m, axis, bias_shrinkage):
+    if m.bias is None:
+        return m.weight.data
+
+    bias = m.bias.data
+    weight = m.weight.data if axis == 0 else m.weight.data.transpose(1,0)
+    weight = deepcopy(weight).view(weight.shape[0], -1)
+    bias = deepcopy(bias).view(-1, 1)
+
+    weight = torch.where(torch.abs(weight) < 1e-8, torch.tensor(1e-8), weight)
```

Comment:
Two things:
- can we reuse the existing small epsilon constant that we already defined, instead of 1e-8?
- Whenever you are creating a new tensor with torch.tensor, you should always be mindful of its device and dtype. For all we know weight could be float16 on GPU, and in that case torch.where would fail. In this case it's a good idea to set device and dtype to the same as weight